.github/workflows/podman-test.yaml

```diff
@@ -23,5 +23,8 @@ jobs:
 
     - name: Run Integration tests
       env:
-        PODMAN_EXEC_NODES: ${{ vars.PODMAN_TEST_EXEC_NODES }}
-      run: make test-integration-podman
+        PODMAN_EXEC_NODES: ${{ secrets.PODMAN_TEST_EXEC_NODES }}
+      run: |
+        make test-integration-podman
+        podman pod ls --format '{{.Name}}' | xargs -I '{}' podman pod inspect '{}'
```

Comment:
To make it easier to analyze the output in the Workflow logs (and decouple it from the integration tests output), can we run these `podman` commands in separate steps instead?
We can run it with an `if: always()` condition so it runs regardless of the status of the `Run Integration tests` previous step.